src/libmongoc/tests/test-mongoc-background-monitoring.c

```diff
@@ -154,18 +166,43 @@ static void
 _server_changed (const mongoc_apm_server_changed_t *event)
 {
    test_fixture_t *tf;
+   const mongoc_server_description_t *old_sd;
    const mongoc_server_description_t *new_sd;
 
    tf = (test_fixture_t *) mongoc_apm_server_changed_get_context (event);
+   old_sd = mongoc_apm_server_changed_get_previous_description (event);
    new_sd = mongoc_apm_server_changed_get_new_description (event);
    bson_mutex_lock (&tf->mutex);
-   TF_LOG (tf, "server changed");
+   TF_LOG (tf,
+           "server changed %s => %s",
+           mongoc_server_description_type (old_sd),
+           mongoc_server_description_type (new_sd));
    tf->observations->sd_type = new_sd->type;
    tf->observations->n_server_changed++;
    mongoc_cond_broadcast (&tf->cond);
    bson_mutex_unlock (&tf->mutex);
 }
 
+#define TV \
+   "{ 'processId': { '$oid': 'AABBAABBAABBAABBAABBAABB' }, 'counter': 1 }"
```

Comment:
Having looked at the tests in this file, I didn't see anything that tests RTT or increasing topology versions. Should either of those be tested here, or is that something covered by spec tests?

With regard to RTT, I'm curious how that factors in given that a server monitor for streamable isMaster would be accompanied by a separate RTT thread -- wouldn't the RTT pings interfere with the mocked auto-responses?